submarine-server/server-core/src/main/resources/org/apache/submarine/database/mappers/ModelVersionMapper.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+-->
+<!DOCTYPE mapper PUBLIC "-//mybatis.org//DTD Mapper 3.0//EN" "http://mybatis.org/dtd/mybatis-3-mapper.dtd">
+<mapper namespace="org.apache.submarine.server.model.database.mappers.ModelVersionMapper">
+  <resultMap id="resultMap" type="org.apache.submarine.server.model.database.entities.ModelVersionEntity">
+    <result column="name" property="name" />
+    <result column="version" property="version" />
+    <result column="creation_time" property="createTime" />
+    <result column="last_updated_time" property="lastUpdatedTime" />
+    <result column="description" property="description" />
+    <result column="user_id" property="userId" />
+    <result column="current_stage" property="currentStage" />
+    <result column="source" property="source" />
+    <result column="run_id" property="runId" />
+    <result column="status" property="status" />
+    <result column="status_message" property="statusMessage" />
```

Comment:
@jeff-901 I am confused with `current_stage` and `status`. What is the difference between these two data? Thanks

Reply:
@KUAN-HSUN-LI  `current_stage` indicates the model is Staging, Production or Archived. `status` shows whether the model is ready.

Reply:
@KUAN-HSUN-LI  `current_stage` indicates the model is Staging, Production or Archived. `status` shows whether the model is ready.

Reply:
@jeff-901  Thanks for your explanation.